src/styles/isomer-template.scss

```diff
@@ -11901,7 +11900,6 @@ html.has-navbar-fixed-top-widescreen {
 }
 
 #key-highlights .col {
```

Comment:
which part of the Isomer UI do I test to see if it looks good?

Reply:
i mentioned in the PR description that 
> cursor will be pointer when hovering over key-highlights

is this unclear ah? will add ss for additional clarity also

Reply:
ps, taking ss removes the cursor so i'm not sure how to show this

Reply:
Can record as 5 second video haha

Reply:
I think it's fine i will just run through your test plan

Reply:
oh also you can do cmd+shift+3 then crop hahaha (for future use)